contents/tutorials/how-to-rename-events.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: How to rename events
+sidebar: Docs
+showTitle: true
+author:
+  - bijan-boustani
```

Comment:
You'll need to add yourself to `authors.json` for this to work! (That's why the build is breaking.)

Your `profile_id` is `31729`